index.html

```diff
@@ -124,8 +125,22 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
                     <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" tabindex="23" required>
                     <hr class="InputSeparation">
                     <textarea name="comment" id="comment" cols="30" rows="5" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here..." tabindex="24" required></textarea>
-                    <button type="submit" class="msjButton" tabindex="25">Get in touch</button>
+                    <p id="validationMsj" class="errorMsj"></p>
+                    <button id="SubmitButton" type="button" class="msjButton" tabindex="25">Get in touch</button>
                 </form>
+                <!--<form id="myForm" class="form_comment">
+                    <input type="text" id="fname" name="fname" placeholder="Full Name" class="input_place input_comment first_name" tabindex="22">
+                    <hr class="InputSeparation">
+                    <br>
+                    <br>
+                    <br>
+                    <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" tabindex="23">
+                    <hr class="InputSeparation">
+                    <textarea name="comment" id="comment" cols="30" rows="5" class="text_comment" placeholder="Write some text here..." tabindex="24"></textarea>
+                    
+                </form>-->
+                <!--<div>Get in touch</div>
+                <button id="SubmitButton" class="msjButton" tabindex="25" onclick="validateForm()">Get in touch</button>-->
```

Comment:
Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Please consider removing 
all the commented codes. :pray: